lib/Segments/SegmentsRepository.php

```diff
@@ -33,6 +33,23 @@ public function getCountsPerType(): array {
     return $countMap;
   }
 
+  public function isNameUnique(string $name, ?int $id): bool {
```

Comment:
Does this method have to be public?

Reply:
How can I call this method from the Controller when it is private/protected?

Reply:
Oh Sorry! My bad! I read the diff as it was one file and thought that it is also in the controller.

Reply:
Don't worry. Thank you, that you have looked at this PR during support rotation